src/base/base_uses.f90

```diff
@@ -12,6 +12,10 @@
                                              timeset,&
                                              timestop
 
+! Check for OpenMP early on - ideally before the compiler fails with a cryptic message.
+#if !defined(_OPENMP)
+   "OpenMP is required. Please add -fopenmp to your Fortran compiler flags."
```

Comment:
Is `#error` bad style in the Fortran code files in CP2K? The current error message gets the point across, but it IMO looks a bit nicer with as `#error`:

```
   17 |    "OpenMP is required. Please add -fopenmp to your Fortran compiler flags."
      |     1
Error: Invalid character in name at (1)
```

vs.

```
   17 | #error  "OpenMP is required. Please add -fopenmp to your Fortran compiler flags."
      |  1~~~~
Error: #error "OpenMP is required. Please add -fopenmp to your Fortran compiler flags."
```

Reply:
The preprocessor employed by Fortran is the C preprocessor in traditional mode, hence no `#error` available and some preprocessor will then choke on the unsupported preprocessor directive rather than on the error message itself (even when disabled). But this will be solved once #1259 lands.